src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -218,23 +305,36 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     const newImage = this.hass.states[this._config.entity]?.attributes
```

Comment:
This needs to be now `attrs.entity_picture_local || attrs.entity_picture` for both old and new.